distribution/docs/src/main/resources/_contents/_catalog-frameworks/catalog-fanout-framework-contents.adoc

```diff
@@ -0,0 +1,71 @@
+
+==== Catalog Fanout Framework 
+
+The Fanout Catalog Framework provides an implementation of the Catalog Framework that acts as a proxy, federating requests to all available sources. 
+All requests are executed as federated queries and resource retrievals, allowing the fanout site to be the sole site exposing the functionality of all of its Federated Sources.
+
+The Fanout Catalog Framework provides the capability to configure ${branding} to be a fanout proxy to other federated sources within the enterprise.
+The Fanout Catalog Framework has no catalog provider configured for it, so it does not allow catalog modifications to take place.
```

Comment:
Did we need to update anything here for the fanout changes @gordocanchola @pklinef ?